scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/cipher/InvalidMacException.java

```diff
@@ -18,25 +18,46 @@
 import java.security.GeneralSecurityException;
 import java.util.Arrays;
 
+import org.eclipse.californium.elements.util.Bytes;
+
 /**
  * This exception is thrown when a MAC validation fails.
  * 
  */
 public class InvalidMacException extends GeneralSecurityException {
 
 	private static final long serialVersionUID = 1L;
+	private static final String DEFAULT_MESSAGE = "MAC validation failed";
 
 	private final byte[] expected;
 	private final byte[] actual;
-	
+
+	/**
+	 * Create generic invalid MAC exception without actual MAC values.
+	 */
+	public InvalidMacException() {
+		this(DEFAULT_MESSAGE);
+	}
+
+	/**
+	 * Create invalid MAC exception without actual MAC values.
+	 * 
+	 * @param msg message
+	 */
+	public InvalidMacException(String msg) {
+		super(msg);
+		this.expected = Bytes.EMPTY;
+		this.actual = Bytes.EMPTY;
```

Comment:
You don't use this constructor is it intended ? (maybe you planned to add a more specific message for padding error above ?)

Reply:
It's used by

```
public InvalidMacException() {
	this(DEFAULT_MESSAGE);
}
```

(even if that may be replaced.)